src/views/template/customer/customerorder.html

```diff
@@ -1,11 +1,16 @@
-<div class="panel panel-default">
-    <div class="panel-heading">
+<li class="list-group-item">
+    <div class="">
         {{ #customer }}
-            {{ name }}
+            <b>{{ name }}</b>
         {{ /customer }}
-        <span class="label label-warning pull-right">{{#l}}Earn{{/l}} {{#order}} {{ orderCost }} {{/order}}</span>
+        <button type="button" class="customer-serve btn btn-default btn-xs" style="float: right;" data-customer="{{ id }}">
```

Comment:
@Rinnerdos this data attribute is how you may obtain the current customer ID.

Reply:
Alright, and how do I code it in the JS? I now have, 
`$("#customer.id").html("<img src=\"src/assets/emojis/angrySmall.png\" alt=\"Smiley\">");`
though that doesn't seem to work. And then there also was something with .attr so that I only have to change the src right?

Reply:
@Rinnerdos look into [this](https://api.jquery.com/data/#data2).

Reply:
Will do.